playbooks/robusta_playbooks/kubectl_enrichments.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import uuid
+
+from hikaru.model.rel_1_26 import Container, PodSpec
+from robusta.api import (
+    RUNNER_SERVICE_ACCOUNT,
+    ExecutionBaseEvent,
+    FileBlock,
+    MarkdownBlock,
+    PodRunningParams,
+    RobustaJob,
+    action,
+)
+from robusta.utils.parsing import format_event_templated_string
+
+IMAGE: str = os.getenv("KUBECTL_IMAGE_OVERRIDE", f"bitnami/kubectl:latest")
+
+
+class KubectlParams(PodRunningParams):
+    """
+    :var kubectl_command: The full kubectl command to run, formatted as a shell command string.
+    :var description: A description of the command ran.
+    :var timeout: The maximum time (in seconds) to wait for the kubectl command to complete. Default is 3600 seconds.
+    """
+
+    command: str = None  # type: ignore
+    description: str = None
+    timeout: int = 3600
+
+
+@action
+def kubectl_command(event: ExecutionBaseEvent, params: KubectlParams):
+    """
+    Runs a custom kubectl command inside a Kubernetes pod using a Job.
+    """
+
+    subject = event.get_subject()
+    formatted_kubectl_command = format_event_templated_string(subject, params.command)
+    
+    logging.debug(f"kubectl_command running '{formatted_kubectl_command}'")
```

Comment:
Please change it to info, I think that this log is something that we alway see since it might have effect for users.

Reply:
we dont want to always have this incase there is sensitive information there

Reply:
In that case is it fine in debug? maybe we can instead log the description?

Reply:
debug should be fine